src/common/kafka/Producer.cpp

```diff
@@ -119,6 +119,8 @@ int Producer::produce(nonstd::span<const std::uint8_t> Buffer,
     XTRACE(KAFKA, DEB, "produce: %s", RdKafka::err2str(resp).c_str());
     stats.produce_fails++;
     return resp;
+  } else {
+    stats.produce_no_errors++;
```

Comment:
The formatting is not correct this line should be indented inside the else block.

Reply:
Sorry, maybe I am blind. What is the indentation error?

Reply:
I' wrong, I was confused on the 
```c++
} else {
```
Everything OK here